applications/FluidDynamicsApplication/python_scripts/cfl_output_process.py

```diff
@@ -0,0 +1,180 @@
+# Importing the Kratos Library
+import KratosMultiphysics
+import KratosMultiphysics.FluidDynamicsApplication as KratosCFD
+
+# other imports
+from KratosMultiphysics.time_based_ascii_file_writer_utility import TimeBasedAsciiFileWriterUtility
+
+from statistics import mean, stdev
+
+
+def Factory(settings, model):
+    if(type(settings) != KratosMultiphysics.Parameters):
+        raise Exception(
+            "expected input shall be a Parameters object, encapsulating a json string")
+
+    return CFLOutputProcess(model, settings["Parameters"])
+
+
+class CFLOutputProcess(KratosMultiphysics.Process):
+    """
+    Auxiliary base class to output total flow forces
+    over obstacles in fluid dynamics problems.
+    A derived class needs to be implemented to be able to use
+    this functionality, as calling the base class alone is not enough.
+    """
+
+    def __init__(self, model, params):
+        """
+        Auxiliary class to output total flow forces over obstacles
+        in fluid dynamics problems for a body fitted model part.
+        """
+        KratosMultiphysics.Process.__init__(self)
+
+        default_settings = KratosMultiphysics.Parameters("""
+            {
+                "model_part_name"      : "",
+                "interval"             : [0.0, 1e30],
+                "cfl_threshold"        : 5.0,
```

Comment:
I understand that this process is just for outputting the local CFL. Then, which is the intention of this `cfl_threshold`. Why do we want to limit it if we just want to output it?

Reply:
The threshold value is not limiting, it just helps for an additional statistics value in the summarize. Many times you would not only be interested in how many elements have CFL above 1.0, but lets say 2.5.

Reply:
Understood. I'd rather call it `cfl_output_lower_bound` or something similar. I find that`cfl_threshold` is a bit misleading.

Reply:
It also does not bound anything, it is just a threshold or limit value (would that be better) for the _Summarize function.

Reply:
or custom limit?

Reply:
> or custom limit?
I'd go for either `cfl_output_limit` or `cfl_output_custom_limit` (the one you like the most). What should be clear is that we are not limiting the CFL, just the output information.